docs/source/resources/troubleshooting.md

```diff
@@ -55,3 +59,21 @@ def _memory_usage(df):
     size += df.index.memory_usage(deep=True)
     return size
 ```
+
+### 2. Initializing a Dask CUDA Cluster
+
+Even if you execute the NVTabular workflow with only a single GPU, using a dask CUDA cluster can optimize your memory usage. If there is no CUDA cluster initialized, NVTabular executes the workflow on workers of a dask cluster. Dask CUDA cluster provides different memory spilling strategies for GPUs and in our experiment, we could resolve OOM errors by initializing a dask CUDA cluster, first. It is easy to provide NVTabular with a dask CUDA cluster by using `LocalCUDACluster`.
```

Comment:
> If there is no CUDA cluster initialized, NVTabular executes the workflow on workers of a dask cluster

I don't think this is quite accurate - if we don't pass a dask client we run the workflow on the default 'synchronous' scheduler inside the current process (instead of on a dask cluster).

Reply:
Thank you. I am sorry, I do not know it that depth :) Does it work to update the sentence with

```
If there is no CUDA cluster initialized, NVTabular's default behaviour is to execute the workflow with a synchronous scheduler inside the current process.
```

Reply:
Updated with Rick's suggestion